server.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": [
+    "./common.json",
+    "./language/es6.json"
+  ],
+  "env": {
+    "node": true
+  },
+  "rules": {
+    "arrow-parens": "error",
+    "no-misleading-character-class": "error",
+    "no-new-require": "error",
+    "no-useless-computed-key": "error",
+    "one-var": 0,
```

Comment:
we should use strings or numbers, but preferably not a mixture: https://eslint.org/docs/user-guide/configuring#configuring-rules

so 0 -> "off"